tools/generate_codemeta.py

```diff
@@ -0,0 +1,112 @@
+import os
+import subprocess
+
+# UPDATE THESE AS NEEDED WITH EACH RELEASE
+release_date = '2020-12-17'
+release_version = '0.22.0'  # major.minor.patch; must include trailing zero
+dependencies = ('Python 3.6', 'NumPy 1.15.4', 'SciPy 1.1')
+# add to these as necessary
+compound_surnames = (
+    'García Alanis',
+    'van Vliet',
+    'De Santis',
+    'Dupré la Tour',
+    'de la Torre',
+    'van den Bosch',
+    'Van Der Donckt',
+    'van der Meer',
+    'van Harmelen',
+    'Visconti di Oleggio Castello'
```

Comment:
Can you make a note in the releases wiki to check and update this if needed?

Reply:
The wiki is already updated (step 7):

> Edit tools/generate_codemeta.py to update the variables release_date, release_version, dependencies (if needed), and compound_surnames (if needed). Then from the tools folder run python/generate_codemeta.py to update the codemeta metadata for this release.

I'll edit it once I've incorporated your other suggestions (e.g., automating version and release date)

Reply:
comments addressed, and wiki has been updated accordingly.